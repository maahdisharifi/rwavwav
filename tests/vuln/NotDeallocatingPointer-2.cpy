void cleanup(int* ptr1):
    free(ptr1)
end

int main():
    int* ptr1 = malloc(sizeof(int) * 10)
    int* ptr2 = malloc(sizeof(int) * 20)
    cleanup(ptr1)
end