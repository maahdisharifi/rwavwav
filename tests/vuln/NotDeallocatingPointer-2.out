Line:7 -> memory not deallocated